feather/server/src/packet_handlers/health.rs

```diff
@@ -0,0 +1,40 @@
+use crate::{ClientId, NetworkId, Position, Server};
+use common::Game;
+use ecs::{Entity, SysResult};
+use protocol::packets::client::ClientStatus;
+use uuid::Uuid;
+
+pub fn handle_client_status(
+    game: &mut Game,
+    server: &mut Server,
+    player_id: Entity,
+    packet: ClientStatus,
+) -> SysResult {
+    match packet {
+        ClientStatus::PerformRespawn => {
+            let client_id = game.ecs.get::<ClientId>(player_id).unwrap();
+            let client = server.clients.get(*client_id).unwrap();
```

Comment:
Can't you `?` these instead of unwrapping?

Reply:
Yea, not sure why I used unwrap there, but returning an error is the better way. Though not sure what error would be appropriate for `server.clients.get` instead of unwrapping if any?